core/services/chainlink/config.go

```diff
@@ -42,8 +42,16 @@ type Config struct {
 	Solana solcfg.TOMLConfigs `toml:",omitempty"`
 
 	Starknet stkcfg.TOMLConfigs `toml:",omitempty"`
+
+	Aptos []*RawConfig `toml:",omitempty"`
 }
 
+// RawConfig is the config used for chains that are not yet public.
```

Comment:
Let's dial this back - we don't actually want any chains here, it will just take some time to remove the existing:
```suggestion
// RawConfig is the config used for chains that are not embedded.
```

Reply:
thanks for the note